--- openmpi-2.1.1/opal/include/opal/sys/x86_64/timer.h~	2017-05-10 15:41:36.000000000 +0000
+++ openmpi-2.1.1/opal/include/opal/sys/x86_64/timer.h	2017-09-17 20:22:34.714118986 +0000
@@ -39,7 +39,7 @@
 {
      unsigned l, h;
 #if !OPAL_ASSEMBLY_SUPPORTS_RDTSCP
-     __asm__ __volatile__ ("cpuid\n\t"
+     __asm__ __volatile__ ("lfence\n\t"
                            "rdtsc\n\t"
                            : "=a" (l), "=d" (h)
                            :: "rbx", "rcx");
@@ -52,7 +52,7 @@
      __asm__ __volatile__ ("rdtscp\n\t"
                            "mov %%edx, %0\n\t"
                            "mov %%eax, %1\n\t"
-                           "cpuid\n\t"
+                           "lfence\n\t"
                            : "=r" (h), "=r" (l)
                            :: "rax", "rbx", "rcx", "rdx");
 #endif
