diff --git a/opal/runtime/opal_progress.c b/opal/runtime/opal_progress.c
index 8c88a32..05e7b55 100644
--- a/opal/runtime/opal_progress.c
+++ b/opal/runtime/opal_progress.c
@@ -254,7 +254,7 @@ opal_progress(void)
          * the processor is oversubscribed - this will result in a best-case
          * latency equivalent to the time-slice.
          */
-        sched_yield();
+        usleep(50);
     }
 #endif  /* defined(HAVE_SCHED_YIELD) */
 }
