--- openmpi-2.1.3/opal/include/opal/sys/x86_64/timer.h~	2017-10-28 20:30:29.000000000 +0000
+++ openmpi-2.1.3/opal/include/opal/sys/x86_64/timer.h	2019-04-18 15:18:03.054339330 +0000
@@ -36,7 +36,7 @@
 opal_sys_timer_get_cycles(void)
 {
      uint32_t l, h;
-     __asm__ __volatile__ ("lfence\n\t"
+     __asm__ __volatile__ (
                            "rdtsc\n\t"
                            : "=a" (l), "=d" (h));
      return ((opal_timer_t)l) | (((opal_timer_t)h) << 32);
