diff --git a/opal/include/opal/sys/x86_64/timer.h b/opal/include/opal/sys/x86_64/timer.h
index a367f77..2250046 100644
--- a/opal/include/opal/sys/x86_64/timer.h
+++ b/opal/include/opal/sys/x86_64/timer.h
@@ -36,7 +36,7 @@ static inline opal_timer_t
 opal_sys_timer_get_cycles(void)
 {
      uint32_t l, h;
-     __asm__ __volatile__ ("lfence\n\t"
+     __asm__ __volatile__ (
                            "rdtsc\n\t"
                            : "=a" (l), "=d" (h));
      return ((opal_timer_t)l) | (((opal_timer_t)h) << 32);
