--- openmpi-2.1.1/orte/tools/orterun/orterun.c~	2017-05-10 15:41:36.000000000 +0000
+++ openmpi-2.1.1/orte/tools/orterun/orterun.c	2017-09-17 20:06:13.678792882 +0000
@@ -664,7 +664,6 @@
         fprintf(stderr, "option to your cmd line. However, we reiterate our strong advice\n");
         fprintf(stderr, "against doing so - please do so at your own risk.\n");
         fprintf(stderr, "--------------------------------------------------------------------------\n");
-        exit(1);
     }
 
     /*
--- openmpi-2.1.1/orte/tools/orte-submit/orte-submit.c~	2017-05-10 15:41:36.000000000 +0000
+++ openmpi-2.1.1/orte/tools/orte-submit/orte-submit.c	2017-09-17 20:06:54.413126778 +0000
@@ -390,7 +390,6 @@
         fprintf(stderr, "option to your cmd line. However, we reiterate our strong advice\n");
         fprintf(stderr, "against doing so - please do so at your own risk.\n");
         fprintf(stderr, "--------------------------------------------------------------------------\n");
-        exit(1);
     }
  
      /*
--- openmpi-2.1.1/orte/tools/orte-dvm/orte-dvm.c~	2017-05-10 15:41:36.000000000 +0000
+++ openmpi-2.1.1/orte/tools/orte-dvm/orte-dvm.c	2017-09-17 20:07:09.210500957 +0000
@@ -198,7 +198,6 @@
         fprintf(stderr, "option to your cmd line. However, we reiterate our strong advice\n");
         fprintf(stderr, "against doing so - please do so at your own risk.\n");
         fprintf(stderr, "--------------------------------------------------------------------------\n");
-        exit(1);
     }
 
     /*
