--- a/opal/mca/base/mca_base_var.c	2019-10-23 14:31:50.658766237 -0700
+++ b/opal/mca/base/mca_base_var.c-new	2019-10-23 16:11:04.743062768 -0700
@@ -409,9 +409,9 @@
     }
 
 #if OPAL_WANT_HOME_CONFIG_FILES
-    asprintf(&mca_base_var_files, "%s"OPAL_PATH_SEP".openmpi" OPAL_PATH_SEP
-             "mca-params.conf%c%s" OPAL_PATH_SEP "openmpi-mca-params.conf",
-             home, ',', opal_install_dirs.sysconfdir);
+    asprintf(&mca_base_var_files, "%s/.openmpi/mca-params.conf,%s/openmpi-mca-params.conf,"
+             "/usr/share/defaults/etc/openmpi/openmpi-mca-params.conf", 
+             home, opal_install_dirs.sysconfdir);
 #else
     asprintf(&mca_base_var_files, "%s" OPAL_PATH_SEP "openmpi-mca-params.conf",
              opal_install_dirs.sysconfdir);
--- a/orte/runtime/orte_mca_params.c-org	2019-10-23 18:19:32.559446641 -0700
+++ b/orte/runtime/orte_mca_params.c	2019-10-23 18:17:29.531440514 -0700
@@ -379,8 +379,12 @@
                                   &orte_default_hostfile);
 
     if (NULL == orte_default_hostfile) {
-        /* nothing was given, so define the default */
-        asprintf(&orte_default_hostfile, "%s/openmpi-default-hostfile", opal_install_dirs.sysconfdir);
+        /* User can override default_hostfile in either /usr/share/defaults/etc/openmpi-mca-params.conf or
+         * /etc/openmpi-mca-params.conf with statement such as: "orte_default_hostfile=<path>/hostfile"
+         *
+         * If nothing was given define the default
+         */
+        asprintf(&orte_default_hostfile, "/usr/share/defaults/etc/openmpi/openmpi-default-hostfile");
         /* flag that nothing was given */
         orte_default_hostfile_given = false;
     } else if (0 == strcmp(orte_default_hostfile, "none")) {

